sycl/test-e2e/Matrix/joint_matrix_out_bounds.cpp

```diff
@@ -10,8 +10,6 @@
 // other triples
 
 // REQUIRES: aspect-ext_intel_matrix
-// XFAIL: arch-intel_gpu_bmg_g21
```

Comment:
After adjustment of sizes, the test started to pass on BMG. So, it was probably failing because non-spec-conformant sizes were used.